README.md

```diff
@@ -59,13 +47,13 @@ $ curl -s -X POST http://kong:8001/<WORKSPACE>/plugins \
   -d config.http_endpoint=http://my-release-kong-collectorapi:5000
 ```
 
-4. Check that collector is reachable by Kong Admin API.
+3. Check that collector is reachable by Kong Admin API.
```

Comment:
What's the purpose of line 37 above, if you're instructing them to check the connection here?